### DISCUSSION TITLE
Welcome to multus-cni Discussions!

### DISCUSSION THREAD

**Discussion by dougbtv** [k8snetworkplumbingwg/multus-cni#646] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emujicad** [comment]
I'd like to know what is the differences between install multus as deamonset using
/deployments/multus-daemonset-thick-plugin.yml as is written in docs/quickstart.md or  /deployments/multus-daemonset.yml in  docs/how-to-use.md

Also what do recommend?
